### PR TITLE
Add column control panel tests

### PR DESCRIPTION
**Pull request by marthacryan**
Adds tests for:
- Changing column dtype
- Adding a filter
- Sorting
- Changing the number type
- Filtering values through the values tab
- Loading summary stats

Adds utilities for:
- opening the column control panel
- changing the dtype
- changing the number type
- checking that column cells have expected values
- checking the control panel's value tab has expected values

### CODE REVIEW

File: tests/streamlit_ui_tests/features/column_control_panel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+const checkValuesTabHasExpectedValues = async (mito: FrameLocator, expectedValues: any[]) => {
+    const values = await mito.locator('.multi-toggle-box').locator('.multi-toggle-box-row').all();
+    for (const valueIndex in values) {
+        const value = values[valueIndex];
+        await expect(value).toHaveText(expectedValues[valueIndex]);
+    }
+};
+
+test.describe('Column Control Panel', () => {
```
[the remark this answers is not in the document]
Comment (aarondr77) [reply tests/streamlit_ui_tests/features/column_control_panel.spec.ts:41]: And if you are on the column summary stats tab, it should remain on that tab. (I've seen users do this a bunch)

File: tests/streamlit_ui_tests/features/column_control_panel.spec.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        await awaitResponse(page);
+        await checkColumnCellsHaveExpectedValues(mito, 0, ['1', '7', '10']);
+    })
+
+    test('Loads summary stats', async ({ page }) => {
+        const mito = await getMitoFrameWithTestCSV(page);
+        await openColumnControlPanel(mito, 'Column1');
+
+        await mito.locator('.control-panel-taskpane-tab', { hasText: 'Stats'}).click();
+        await expect(mito.locator('.column-describe-table-row', { hasText: 'mean' })).toContainText('5.5')
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/features/column_control_panel.spec.ts:123]: The column summary graph is loading for me on JupyterLab safari, can you double check that its not working on streamlit and then open an issue if so

**Review by aarondr77 — COMMENTED** [review]
Super lovely. A few more test ideas and then LGTM

File: tests/streamlit_ui_tests/features/column_control_panel.spec.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+const checkColumnCellsHaveExpectedValues = async (mito: FrameLocator, columnIndex: number, values: any[]) => {
+    const cells = await mito.locator(`.mito-grid-cell[mito-col-index="${columnIndex}"]`).all()
+    await expect(mito.locator('.index-header-container')).toHaveCount(values.length);
+    for (const cellIndex in values) {
+        const cell = cells[cellIndex];
+        await expect(cell).toHaveText(values[cellIndex]);
+    }
+}
+
+const checkValuesTabHasExpectedValues = async (mito: FrameLocator, expectedValues: any[]) => {
```
[the remark this answers is not in the document]
Comment (marthacryan) [reply tests/streamlit_ui_tests/features/column_control_panel.spec.ts:22]: Not that I see?

**Pull request merged**